src/config/dynamic-theme-fixes.config

```diff
@@ -585,6 +585,27 @@ body[style] {
 
 ================================
 
+bestbuy.ca
+
+INVERT
+.centerContainer
+
+CSS
+[class^="facetName"], [class^="title"], [class^="sortLabel"], [class^="headerText"], [class^="subTitle"] {
+    filter:invert(100%) hue-rotate(180deg) contrast(90%) !important;
+}
+.backgroundContainer_ryHAa {
```

Comment:
Where can I find this element?

Reply:
https://www.bestbuy.ca/en-ca/category/gaming-monitors/474835